<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Race progress</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Reload</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QTableView" name="runnersTableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Race connections</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Racetiming website</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="websiteLineEdit">
          <property name="text">
           <string>racetiming.happyforever.com</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>API key</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="apiKeyLineEdit">
          <property name="text">
           <string>12345</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="raceConnectionsConnectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Available races</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QComboBox" name="availableRacesComboBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="getRunnersPushButton">
             <property name="text">
              <string>Get runners</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>RFID reader connections</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QComboBox" name="comboBox_6"/>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>Start/Lap</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Finish</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>M6E Nano</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_4">
          <item>
           <property name="text">
            <string>Custom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>M6E Nano</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="comboBox_3"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Connection</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="comboBox_5">
          <item>
           <property name="text">
            <string>Start/Lap</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Finish</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Function</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Events</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Enter custom event</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="3">
            <widget class="QSpinBox" name="msecsSpinBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="raceIDSpinBox"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Runnerid</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Event Type</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Raceid</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Timestamp</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QDateTimeEdit" name="timestampDateTimeEdit"/>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>msecs</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QComboBox" name="eventTypeComboBox"/>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="runnerIDSpinBox"/>
           </item>
           <item row="1" column="5">
            <widget class="QPushButton" name="insertEventPushButton">
             <property name="text">
              <string>Insert</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>898</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
